recipes/hybridcontents/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hybridcontents" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: af8e3f6886533748761e41bd12690a9c4854e2a12baae2d49f5b72258f3138e4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - notebook >=4.0
```

Comment:
Usually, having `pip` and `python` is enough in the `host` section. Can you please try removing `notebook`, `six` and `requests` from here, and leaving them in `run`?

Reply:
Worked!